clonelists/Atari - 2600 (No-Intro).json

```diff
@@ -181,7 +181,8 @@
 			"group": "I.Q. 180",
 			"titles": [
 				{"searchTerm": "I.Q. 180"},
-				{"searchTerm": "I.Q. Memory Teaser"}
+				{"searchTerm": "I.Q. Memory Teaser"},
+				{"searchTerm": "IQ 180"}
```

Comment:
Nice catch! It looks like [Stella](https://stella-emu.github.io/) has a manufacturer field for `IQ180`, whereas for `I.Q. 180` it's empty. This makes me wonder if `IQ180` should be the selected title for USA priority region orders?

Because `I.Q. 180` and `IQ 180` are from the same region (USA), you need to set a priority to make sure Retool selects the correct title, like so:

```json
{"searchTerm": "IQ 180"},
{"searchTerm": "I.Q. Memory Teaser"},
{"searchTerm": "I.Q. 180", "priority": 2}
```

Reply:
That makes sense!